packages/workspace/src/core/project-graph/project-graph.ts

```diff
@@ -38,30 +39,85 @@ import {
   buildWorkspaceProjectNodes,
 } from './build-nodes';
 
+export const NEXT_GRAPH_VERSION = '4.0';
+export const CURRENT_GRAPH_VERSION = '3.0';
+
 /**
  * Synchronously reads the latest cached copy of the workspace's ProjectGraph.
  * @throws {Error} if there is no cached ProjectGraph to read from
+
+ * @param {string} projectGraphVersion Version to map ProjectGraph to
+ * @returns {ProjectGraph}
  */
-export function readCachedProjectGraph(): ProjectGraph {
+export function readCachedProjectGraph(
+  projectGraphVersion?: string
```

Comment:
I think this should have a default value set to '3.0'. That was the assumed value before.